File: Components/BlogML/Enumerators/BlogPostTypes.vb
Imports System.Xml.Serialization

Namespace BlogML
 Public Enum BlogPostTypes As Short
  <System.Xml.Serialization.XmlEnumAttribute("normal")> _
  Normal = 1
  <System.Xml.Serialization.XmlEnumAttribute("article")> _
  Article = 2
 End Enum
End Namespace

File: Server/Core/BlogML/Enumerators/BlogPostTypes.cs

namespace DotNetNuke.Modules.Blog.Core.BlogML
{
  public enum BlogPostTypes : short
  {
    [System.Xml.Serialization.XmlEnum("normal")]
    Normal = 1,
    [System.Xml.Serialization.XmlEnum("article")]
    Article = 2
  }
}